### ISSUE TITLE
Multiplayer

### ISSUE THREAD

**Issue by KayTaneJupiter** [sharkwouter/minigalaxy#177]
Does not do GOG multiplayer?

Tested with Grim Dawn (Wine). Game reports that it is not connected to GOG Galaxy for multiplayer.

**Comment by reidkader** [comment]
I dont think so. Minigalaxy is FOSS GOG client with limited functionality. If You want to play multiplayer You must stick to official GOG Galaxy 2.0 run by wine. For now there's no other way.

**Comment by NinCollin** [comment]
The [Comet Daemon](https://gitlab.com/Yepoleb/comet/-/tree/master) is a simple program which supports the multiplayer service for GOG. Since both Comet and this project use GPLv3, couldn't Comet eventually be forked/integrated into Minigalaxy? Obviously it would still require a bit of work to smoothly integrate the two, but multiplayer support wouldn't have to be written from scratch.

**Comment by ThisNekoGuy** [comment]
[It's a planned Heroic Launcher feature to add support for this](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher#planned-features), so I hope this does get attention for Minigalaxy, personally.
